lua/nightfox/init.lua

```diff
@@ -5,6 +5,23 @@ end
 
 local config = require("nightfox.config")
 
+local function read_file(filepath)
+  local file = io.open(filepath, "r")
+  if file then
+    local content = file:read()
+    file:close()
+    return content
+  end
+end
+
+local function write_file(filepath, content)
+  local file = io.open(filepath, "w")
```

Comment:
This should be

```lua
local file = io.open(filepath, "wb")
```

Or it will error on windows

Reply:
Ah ok thanks will update it